### PR TITLE
Problem with multi-server mfiles 

### PR DESCRIPTION
**Pull request by barbyware**
HI,

In case of multi-server environment you have to use cookie based sessions instead of authentication tokens:
https://m-files.com/APIs/REST-API/Authentication/#cookie-based-sessions

The solution is easy, create a request.Session and I use this change in all request.post call

### CODE REVIEW

**Comment by barbyware** [comment]
In case of multi-server environment you have to use cookie based sessions instead of authentication tokens:
https://developer.m-files.com/APIs/REST-API/Authentication/#cookie-based-sessions

File: mfiles/client.py [hunk 1]
```diff
@@ -99,7 +99,8 @@ def login(self, server=None, user=None, password=None, vault=None):
                            "Password": self.password,
                            "VaultGuid": self.vault})
         request_url = self.server + "server/authenticationtokens"
-        response = requests.post(request_url, data=auth)
+        self.request = requests.Session()
```
Comment (afcmrp) [inline mfiles/client.py]: This should be set in __init__ and I think a better name is self.session

File: setup.py [hunk 2]
```diff
@@ -11,13 +11,13 @@
...
     packages=find_packages(),
-    version='0.5.2',
+    version='0.5.3',
     license='MIT',
     description='M-Files API wrapper',
     long_description=PYPI_DESCRIPTION,
     author='Emil Hjelm',
     author_email='emil.hjelm@climeon.com',
-    url='https://github.com/afcmrp/python-mfiles',
+    url='https://github.com/barbyware/python-mfiles',
```
Comment (afcmrp) [inline setup.py]: This url should not change

**Review by afcmrp — CHANGES_REQUESTED** [review]
Nicely done, left a few comments that should be resolved before merge.

File: setup.py (hunk 2 above)
Comment (barbyware) [reply setup.py]: OK, you need that I made the changes or could be made yo

**Pull request closed without merging**